config/routes/api_entreprise.rb

```diff
@@ -8,16 +8,18 @@
     end
   end
 
+  post '/auth/api_gouv_entreprise', as: :login_api_gouv
```

Comment:
Histoire d'être consistant je ferai ça:

```suggestion
  post '/auth/api_gouv_entreprise', as: :login_api_gouv_entreprise
```